[2311] Italy 8.33kHz

Unya12 opened the pull request:
Fixes #152 and #178 

data from test li.json with new sectors, new colours and 8.33hz spacing

Unya12 committed:
migrating from test data

data from test li.json with new sectors, new colours and 8.33hz spacing

lennycolton reviewed (changes requested):
- None of the sectors are correctly assigned to groups at the moment. There should be groups for each enroute centre (e.g. Roma or Padova) and groups for APP/TWR units (e.g. Approach, Arrival, Departure, Tower, etc.).
- The `callsigns` section doesn't need a `CTR` entry - this is used only for assigning callsigns to undefined airport-associated positions (e.g. APP or TWR for a minor field with no airspace or topdown).

Unya12 committed:
Groups and Callsigns mod

Hopefully fixed the groups and removed the CTRs in the "callsigns". Also removed eurocontrol positions from sector ownership.

lennycolton committed:
Fix LI groups
